projects/openapi-cli/src/captures/capture-storage.ts

```diff
@@ -0,0 +1,29 @@
+import Path from 'path';
+import * as fs from 'fs-extra';
+import { InputErrors } from '../commands/reporters/feedback';
+import os from 'os';
+const tmpDirectory = os.tmpdir();
+import crypto from 'crypto';
+
+export async function captureStorage(
+  filePath: string
+): Promise<[boolean, string]> {
+  const resolvedFilepath = Path.resolve(filePath);
+  const openApiExists: boolean = await fs.pathExists(resolvedFilepath);
+
+  const specPathHash = crypto
+    .createHash('md5')
+    .update(resolvedFilepath)
+    .digest('hex');
+
+  const trafficDirectory = Path.join(
+    tmpDirectory,
+    'optic',
+    'captures',
+    specPathHash
+  );
```

Comment:
Traffic is stored in `/tmp` by default to make it easier to use the CLI. You don't need to output hars, remember the name, then delete them

Reply:
I noticed this watching your demo video, this is really nice! I expect we'll need to let them override this default at some point.

Reply:
You can output to a file by passing `-o file.har` to `capture` 
And verify can take a file in (to override the tmp dir) the same way `--har file.har` 

Do you think we need more? Or just to document those options well?